src/product.hpp

```diff
@@ -5,50 +5,23 @@
  * Pin Definitions
  *****************************************************************************/
 /**
- * USB Power Detection Pin
+ * USB Power Detection Pin TODO
  */
 #define SF_USB_PWR_DETECT_PIN   45
 /**
- * Water Detect Enable Pin
+ * Battery Charging Status Pin 
  */
-#define WATER_DETECT_EN_PIN   B4
-/**
- * Water Detect Pin
- */
-#define WATER_DETECT_PIN      B5
-/**
- * @brief Manufacturing Water Detect Pin
- * 
- */
-#define WATER_MFG_TEST_EN     A3
-/**
- * Battery Charging Status Pin
- */
-#define STAT_PIN              A5
+#define STAT_PIN              50
 /**
  * Pin for the Battery Status LED
  */
-#define STAT_LED_PIN          A4
-/**
- * Pin for the Water Detect State LED
- */
-#define LED_PIN               C2
+#define STAT_LED_PIN          40
```

Comment:
This doesn't seem to work!  I don't record any movement on pin 40 (A5) on the TSoM!